.github/CODEOWNERS

```diff
@@ -19,157 +19,183 @@
 # This should make it easy to add new rules without breaking existing ones.
 
 ######## API landing page ###########
-/api/** @dotnet/docs
+/api/ @dotnet/docs
```

Comment:
I think you still want ** on these items, otherwise it won't include child folders+items